### PR TITLE
feat: Creating hackathon Participants table

### PR DESCRIPTION
**Pull request by divanshu3451**

### CODE REVIEW

File: src/db/schema/hackathon.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  boolean,
+  smallint,
+} from "drizzle-orm/pg-core";
+import { user } from "./user";
+import { timestamps } from "./schema.helper";
+import { relations } from "drizzle-orm";
+import { hackathonParticipant } from "./hackathonParticipant";
+
+export const hackathon = pgTable("Hackathons", {
+  id: integer().primaryKey().generatedAlwaysAsIdentity(),
```
Comment (PrakashGit205) [inline src/db/schema/hackathon.ts]: We should use serial instead of integer here.

Comment (divanshu3451) [reply src/db/schema/hackathon.ts]: Agreed...

File: src/db/schema/hackathonParticipant.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  pgTable,
+  timestamp,
+  varchar,
+} from "drizzle-orm/pg-core";
+import { hackathon } from "./hackathon";
+import { user } from "./user";
+import { relations } from "drizzle-orm";
+
+export const hackathonParticipant = pgTable("HackathonParticipants", {
+  id: integer().primaryKey().generatedAlwaysAsIdentity(),
```
Comment (PrakashGit205) [inline src/db/schema/hackathonParticipant.ts]: Same here

Comment (divanshu3451) [reply src/db/schema/hackathonParticipant.ts]: addressed

File: src/db/schema/hackathon.ts (hunk 1 above)
Comment (divanshu3451) [reply src/db/schema/hackathon.ts]: addressed

**Review by PrakashGit205 — DISMISSED** [review]

**Comment by PrakashGit205** [comment]
LGTM

**Review by PrakashGit205 — APPROVED** [review]

**Pull request merged**